Tests/Realm.Tests/Database/RealmDictionaryTests.cs

```diff
@@ -1021,13 +1056,13 @@ public void CanBeQueried()
                 _realm.Add(second);
             });
 
-            var equals = _realm.All<DictionariesObject>().Filter($"{nameof(StringDictionary)}.keys == 'a'");
+            var equals = _realm.All<DictionariesObject>().Filter($"{nameof(StringDictionary)}.@keys == 'a'");
```

Comment:
Why the @?

Reply:
I believe it has to do with NSPredicate-like syntax to query on collections.
@nirinchev is this correct?

Reply:
Yeah - the Core team changed the syntax for referencing the keys of a dictionary.